src/main/java/com/google/sps/data/Coordinates.java

```diff
@@ -22,7 +22,7 @@ public Coordinates() {
     this.lng = -1;
     this.lat = -1;
   }
-
+  
```

Comment:
There should not be any "trailing" white space, a blank line should be entirely blank.